### ISSUE TITLE
Make the score and player panel look better

### ISSUE THREAD

**Issue by nikopetr** [nikopetr/Memory-Game#1]

**Comment by josephAttia** [comment]
Is this Issue still open? I could work on it

**Comment by nikopetr** [comment]
> Is this Issue still open? I could work on it

Of course, go for it! The code is well commented so take a look at it in case you need help. Make sure to include comments and have the code as tidy as possible when you make changes. Let me know if you need anything!

**Comment by josephAttia** [comment]
I submitted a pull request

**Comment by nikopetr** [comment]
> I submitted a pull request

Thank you, going to check it and give you feedback soon
